packages/kitchen-sink/src/examples/data-converter.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
+
+interface DataConverterState {
+inputUnit: DataUnit;
+outputUnit: DataUnit;
+value: number;
+convertedValue: number;
+}
+
+enum DataUnit {
+Byte = "Byte",
+Kilobyte = "KB",
+Megabyte = "MB",
+Gigabyte = "GB",
+Terabyte = "TB",
+Petabyte = "PB",
+}
+
+const DataConverter: React.FC = () => {
```

Comment:
Your component doesn't use Million though. Can you maybe check this out?  Use the `block` function from Millon